sky/resources.py

```diff
@@ -45,12 +45,13 @@ class Resources:
     # 1. Increment the version. For backward compatibility.
     # 2. Change the __setstate__ method to handle the new fields.
     # 3. Modify the to_config method to handle the new fields.
-    _VERSION = 6
+    _VERSION = 7
 
     def __init__(
         self,
         cloud: Optional[clouds.Cloud] = None,
         instance_type: Optional[str] = None,
+        cpu: Union[None, int, float, str] = None,
```

Comment:
Should we update / add a TODO in less_demanding_than() / is_same_resources()?

Reply:
reminder

Reply:
For `is_same_resources`, we don't need to do anything in this PR, because it's not reachable.

For ` less_demanding_than()`, it does not make sense to compare the user-specified `Resources` against the existing cluster's `Resources` in terms of `cpus`, because the latter one always has the `cpus` field set to None.

Reply:
Thanks. For code gardening purpose, let's remove `is_same_resources` in this PR too?

Reply:
Removed.